minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java

```diff
@@ -211,6 +211,16 @@ public void stopFS() throws Exception {
     stopMasters();
   }
 
+  /**
+   * Formats the master and then restarts it. This is useful if a fresh state is desired, for
```

Comment:
Javadoc sounds like the formatting happens before the stopping

Reply:
Updated, thanks.